packages/@lwc/engine-dom/src/renderer.ts

```diff
@@ -5,314 +5,8 @@
-}
-
-function getClassList(element: Element): DOMTokenList {
-    return element.classList;
-}
-
-function setCSSStyleProperty(
-    element: Element,
-    name: string,
-    value: string,
-    important: boolean
-): void {
-    // TODO [#0]: How to avoid this type casting? Shall we use a different type interface to
-    // represent elements in the engine?
-    (element as HTMLElement | SVGElement).style.setProperty(
-        name,
-        value,
-        important ? 'important' : ''
-    );
-}
-
-function getBoundingClientRect(element: Element): DOMRect {
-    return element.getBoundingClientRect();
-}
-
-function querySelector(element: Element, selectors: string): Element | null {
-    return element.querySelector(selectors);
-}
-
-function querySelectorAll(element: Element, selectors: string): NodeList {
-    return element.querySelectorAll(selectors);
-}
-
-function getElementsByTagName(element: Element, tagNameOrWildCard: string): HTMLCollection {
-    return element.getElementsByTagName(tagNameOrWildCard);
-}
-
-function getElementsByClassName(element: Element, names: string): HTMLCollection {
-    return element.getElementsByClassName(names);
-}
-
-function getChildren(element: Element): HTMLCollection {
-    return element.children;
-}
-
-function getChildNodes(element: Element): NodeList {
-    return element.childNodes;
-}
-
-function getFirstChild(element: Element): Node | null {
-    return element.firstChild;
-}
-
-function getFirstElementChild(element: Element): Element | null {
-    return element.firstElementChild;
-}
-
-function getLastChild(element: Element): Node | null {
-    return element.lastChild;
-}
-
-function getLastElementChild(element: Element): Element | null {
-    return element.lastElementChild;
-}
-
-function isConnected(node: Node): boolean {
-    return node.isConnected;
-}
-
-function assertInstanceOfHTMLElement(elm: any, msg: string) {
-    assert.invariant(elm instanceof HTMLElement, msg);
-}
-
-const HTMLElementExported = HTMLElementConstructor as typeof HTMLElement;
-
-export const renderer = {
-    isNativeShadowDefined,
-    isSyntheticShadowDefined,
-    HTMLElementExported,
-    insert,
-    remove,
-    cloneNode,
-    createFragment,
-    createElement,
-    createText,
-    createComment,
-    nextSibling,
-    attachShadow,
-    getProperty,
-    setProperty,
-    setText,
-    getAttribute,
-    setAttribute,
-    removeAttribute,
-    addEventListener,
-    removeEventListener,
-    dispatchEvent,
-    getClassList,
-    setCSSStyleProperty,
-    getBoundingClientRect,
-    querySelector,
-    querySelectorAll,
-    getElementsByTagName,
-    getElementsByClassName,
-    getChildren,
-    getChildNodes,
-    getFirstChild,
-    getFirstElementChild,
-    getLastChild,
-    getLastElementChild,
-    isConnected,
-    insertStylesheet,
-    assertInstanceOfHTMLElement,
-    defineCustomElement,
-    getCustomElement,
-};
+export const renderer = rendererFactory(globalThis, insertStylesheet);
```

Comment:
how would locker be able to import `insertStylesheet`?

Reply:
We discussed doing it in a follow up PR.

Reply:
We discussed doing it in a follow up PR.

Reply:
Decided to just do it in this PR.

Reply:
@caridy We had talked about doing something like 
```
rendererFactory.static.createStyleInsertion(src) {
    // ...
}
```
I figured all of that is not required. We can just do 
```
import { renderer, rendererFactory } from 'lwc';
const sandboxedRendererFactory = evaluateInSandbox(rendererFactory.toString());
const sandboxedRenderer = sandboxedRendererFactory(globalThis);
sandboxedRenderer.__proto__ = renderer;
```
The sandboxedRenderer will inherit all missing properties from `renderer`.